resources/web/wwi/Animation.js

```diff
@@ -1,27 +1,23 @@
-/* global DefaultUrl, TextureLoader */
 'use strict';
+import DefaultUrl from './DefaultUrl.js';
+import {requestFullscreen, exitFullscreen, onFullscreenChange} from './fullscreen_handler.js';
 
-class Animation { // eslint-disable-line no-unused-vars
+export default class Animation {
   constructor(url, scene, view, gui, loop) {
     this.url = url;
     this.scene = scene;
     this.view = view;
     this.gui = typeof gui === 'undefined' || gui === 'play' ? 'real_time' : 'pause';
     this.loop = typeof loop === 'undefined' ? true : loop;
     this.sliding = false;
-    this.onReady = null;
   };
 
   init(onReady) {
     this.onReady = onReady;
-    var xmlhttp = new XMLHttpRequest();
-    xmlhttp.open('GET', this.url, true);
-    xmlhttp.overrideMimeType('application/json');
-    xmlhttp.onreadystatechange = () => {
-      if (xmlhttp.readyState === 4 && xmlhttp.status === 200)
-        this._setup(JSON.parse(xmlhttp.responseText));
-    };
-    xmlhttp.send();
+    fetch(this.url)
+      .then(response => response.json())
+      .then(data => this._setup(data))
+      .catch((error) => console.log(error));
```

Comment:
Here you changed the way the URL is retrieved.
This could explain why local files can no longer be opened (at least on Chrome)